update_test.go

```diff
@@ -579,6 +579,106 @@ func TestUpdate(t *testing.T) {
 				},
 			},
 		},
+		{ // 13 - package/..., missing packages
+			vendor: true,
+			cwd:    "C",
+			args:   []string{"D/..."},
+			start: []*node{
+				{
+					"D",
+					"",
+					[]*node{
+						{"A/main.go", pkg("A") + decl("D1"), nil},
+						{"B/main.go", pkg("B") + decl("D1"), nil},
+						{"+git", "D1", nil},
+					},
+				},
+				{
+					"C",
+					"",
+					[]*node{
+						{"main.go", pkg("main", "D/A"), nil},
+						{"Godeps/Godeps.json", godeps("C", "D/A", "D1", "D/B", "D1"), nil},
+						{"vendor/D/A/main.go", pkg("A") + decl("D1"), nil},
+						{"vendor/D/B/main.go", pkg("B") + decl("D1"), nil},
+						{"+git", "", nil},
+					},
+				},
+				{"D",
+					"",
+					[]*node{
+						{"A/main.go", pkg("A") + decl("D2"), nil},
+						{"B", "(rm)", nil},
+						{"+git", "D2", nil},
+					},
+				},
+			},
+			want: []*node{
+				{"C/vendor/D/A/main.go", pkg("A") + decl("D2"), nil},
+				{"C/vendor/D/B/main.go", "(absent)", nil},
+				{"C/vendor/D/E/main.go", "(absent)", nil},
+			},
+			wdep: Godeps{
+				ImportPath: "C",
+				Deps: []Dependency{
+					{ImportPath: "D/A", Comment: "D2"},
+				},
+			},
+		},
+		{ // 14 - Update package A, but not package B, which is missing from $GOPATH
```

Comment:
merging now. I'll look at cleaning up UX next week. That may actually be harder. ;-)